templates/all-services/all-services-en.html

```diff
@@ -0,0 +1,738 @@
+			<li><a href="https://www.canada.ca/en/employment-social-development/programs/ei/ei-list/ei-roe.html">Record of employment (ROE)</a></li>
+			<li><a href="https://www.canada.ca/en/health-canada/services/health-cards.html">Provincial and territorial health cards</a></li>
+		</ul>
+	</div>
+</section>
+
+<section class="pb-3">
+	<div class="container">
+		<h2 class="h3 mb-4">Most requested services</h2>
+		<ul class="list-unstyled colcount-md-3 small bold-content lst-spcd-2 fnt-hdng">
+			<li><a href="https://www.canada.ca/en/government/sign-in-online-account.html">Sign in to an account</a></li>
+			<li><a href="https://www.canada.ca/en/government/change-address.html">Change your address</a></li>
+			<li><a href="https://www.canada.ca/en/public-services-procurement/services/payments-to-from-government/direct-deposit.html">Set up or change your direct deposit</a></li>
+			<li><a href="https://www.canada.ca/en/government/grants-funding.html">Find grants and funding</a></li>
+			<li><a href="https://www.canada.ca/en/services/jobs/opportunities.html">Find a job</a></li>
+			<li><a href="https://apps.cra-arc.gc.ca/ebci/rhpd/beta/entry">Payroll Deductions Online Calculator</a></li>
+		</ul>
+	</div>
+</section>
+
+<section class="bg-light pb-3">
+	<div class="container">
+		<h2 class="h3 mb-4">Supporting you through life's transitions</h2>
+		<ul class="list-unstyled colcount-md-3 small bold-content lst-spcd-2 fnt-hdng">
+			<li><a href="https://www.canada.ca/en/services/death.html">What to do when someone dies</a></li>
+			<li><a href="https://www.canada.ca/en/services/retirement.html">Learn and plan for your retirement</a></li>
+		</ul>
+	</div>
+</section>
+
+<div class="container">
+	<h2 class="mt-4">Access services by topic</h2>
+
+	<details class="mt-4">
+		<summary>Immigration and citizenship</summary>
+		<section class="bg-info row pt-3 pb-2">
+			<div class="col-md-3">
+				<h3 class="mt-0"><span class="fas fa-exclamation-circle text-info mr-2" aria-hidden="true"></span>Information on</h3>
+			</div>
+			<div class="col-md-9">
+				<ul class="colcount-md-2 small bold-content lst-spcd fnt-hdng">
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/refugees/afghanistan.html">Afghanistan</a></li>
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/haiti-2024.html">Haiti</a></li>
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/iran.html">Iran</a></li>
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/israel-west-bank-gaza-2023.html">Israel, the West Bank and the Gaza Strip</a></li>
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/lebanon-2023.html">Lebanon</a>&nbsp;&nbsp;<span class="label label-primary">Updated</span></li>
```

Comment:
Don't use non-breaking space for layout, use padding or margin instead. This fail SC 1.3.1

```suggestion
					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/lebanon-2023.html">Lebanon</a> <span class="label label-primary">Updated</span></li>
```
If it is no avoid wrapping, then an such CSS utility should be used instead by adding `span` element